site/docs/guide/deployment-types.md

```diff
@@ -96,9 +96,9 @@ ______________                                   _____________
 
 > Note that in reality, no connection would be kept open for hours.
 > Long polling requests have a default timeout of 30 seconds (in order to avoid a number of [technical problems](https://tools.ietf.org/id/draft-loreto-http-bidirectional-07.html#timeouts)).
-> If no new messages are returned after this period of time, then the request will be cancelled and resent—but the general concept stays the same.
+> If no new messages are returned after this period, the request will be canceled and resent—but the general concept stays the same.
 
-Using long polling, you don't need to spam Telegram's servers, and still you get new messages immediately!
+You don't need to spam Telegram's servers using long polling, and you still get new messages immediately!
```

Comment:
Please revert the former change. It's useful to stick with the convention of listing the condition first when examining all cases.